src/client/app/component/LoginForm/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './style.scss';
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div className="LoginBox">
+        <p className="LoginBoxTittle">Log in</p>
+        <form className="LoginForm" >
+          <input type="email" name="email" required placeholder="E-mail" />
+          <input type="password" name="password" required
+            placeholder="Password"/>
+          <input type="submit" value='login'/>
```

Comment:
Do not mix single quotes and double quotes. Use **double quotes** in HTML and JSX